sql/src/test/java/io/crate/analyze/InsertFromValuesAnalyzerTest.java

```diff
@@ -928,7 +928,7 @@ public void testInsertFromValuesWithOnConflictAndNestedColumn() throws Exception
                       "  primary key (o2['p'])" +
                       ")")
             .build();
-        String insertStatement = "insert into nested_on_conflict (o, o2) values ({c=1}, {p=1}) on conflict (o2.p) do update set k = 1";
+        String insertStatement = "insert into nested_on_conflict (o, o2) values ({c=1}, {p=1}) on conflict (\"o2.p\") do update set k = 1";
```

Comment:
The column name needs to be quoted here since it is treated as qualified name. I hope this makes sense.